src/cargo-fmt/main.rs

```diff
@@ -110,16 +122,53 @@ fn execute() -> i32 {
         handle_command_status(format_crate(
             verbosity,
             &strategy,
-            opts.rustfmt_options,
+            rustfmt_args,
             Some(&manifest_path),
         ))
     } else {
-        handle_command_status(format_crate(
-            verbosity,
-            &strategy,
-            opts.rustfmt_options,
-            None,
-        ))
+        handle_command_status(format_crate(verbosity, &strategy, rustfmt_args, None))
+    }
+}
+
+fn convert_message_format_to_rustfmt_args(
+    message_format: String,
```

Comment:
Would you mind changing this from `String` to `&str`? I think that in Rust it is usually better to use `&str` over `String` unless you are modifying or acquiring the ownership of that string.

Reply:
Done!